src/test/java/org/broadinstitute/hellbender/IntelInflaterIntegrationTest.java

```
@@ -0,0 +1,91 @@
+        }
+
+        Assert.assertTrue(new IntelInflater().load(null), "IntelInflater shared library was not loaded. " +
+                "This could be due to a configuration error, or your system might not support it.");
+    }
+
+    @Test
+    public void inflateInflateWithIntelInflater() {
+        if (!isIntelInflaterSupported()) {
+            throw new SkipException("IntelInflater not available on this platform");
+        }
+
+        // create buffers and random input
+        final int LEN = 64 * 1024;
+        final byte[] input = new RandomDNA().nextBases(LEN);
+        final byte[] compressed = new byte[2 * LEN];
+        final byte[] result = new byte[LEN];
+
+        final IntelInflaterFactory intelInflaterFactory = new IntelInflaterFactory();
+
+        for (int i = 0; i < 10; i++) {
+            // create deflater with compression level i
+            final Deflater deflater = new Deflater(i, true);
+
+            // setup deflater
+            deflater.reset();
+            deflater.setInput(input);
+            deflater.finish();
+
+            // compress data
+            int compressedBytes = 0;
+            while (!deflater.finished()) {
+                compressedBytes = deflater.deflate(compressed, 0, compressed.length);
+            }
+            deflater.end();
+
+            // log results
+            log.info("%d bytes compressed to %d bytes : %2.2f%% compression\n",
+                    LEN, compressedBytes, 100.0 - 100.0 * compressedBytes / LEN);
+
+            // decompress and check output == input
+            Inflater inflater = intelInflaterFactory.makeInflater(true);
+            try {
+                inflater.setInput(compressed, 0, compressedBytes);
+                inflater.inflate(result, 0, result.length);
+                inflater.end();
+            } catch (java.util.zip.DataFormatException e) {
+                e.printStackTrace();
```

Should we be trapping this exception like this? Don't we want to let the exception propagate so that the test will faill in this case?